user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/General/Maps.md

```diff
@@ -115,3 +115,19 @@ The following actions are available:
 
   > [!TIP]
   > For more information about the different types of overlays, see [Layer types](xref:Layer_types#layers-of-sourcetype-overlay)
+
+## Zooming
+
+Zooming functionalities are available for the maps component.
+
+- From DataMiner 10.3.0 [CU18]/10.4.0 [CU6]/10.4.9 onwards<!--RN 40017-->:
+
+  - To zoom in, press CTRL while scrolling up.
+
+  - To zoom out, press CTRL while scrolling down.
+
+- Prior to DataMiner 10.3.0 [CU18]/10.4.0 [CU6]/10.4.9:
```

Comment:
Zooming was not mentioned yet on the maps component page, but I assume zooming in/out used to work by using the scroll wheel on its own instead of pressing CTRL at the same time. Let me know if this isn't correct, @Robin-Debel.

Reply:
This is correct, previously no ctrl was needed to zoom. With this change it now is.